exercises/isogram/src/example.c

```diff
@@ -0,0 +1,25 @@
+#include "isogram.h"
+#include <ctype.h>
+#include <stddef.h>
+
+bool isIsogram(const char phrase[])
+{
+   long usedLetters = 0;
```

Comment:
This only needs to be 32-bit, how about using `uint32_t` to both guarantee that it will always be big enough and also to make sure we aren't using more memory than necessary?